cfme/tests/services/test_service_rbac.py

```diff
@@ -0,0 +1,122 @@
+import fauxfactory
+import pytest
+
+from cfme.infrastructure.provider.virtualcenter import VMwareProvider
+from cfme.services.service_catalogs import ServiceCatalogs
+
+pytestmark = [
+    pytest.mark.usefixtures('uses_infra_providers'),
+    pytest.mark.usefixtures('setup_provider_modscope'),
```

Comment:
We shouldn't use this fixture anymore, @psav is this correct?

Reply:
Required: so here we need to use setup_provider instead of setup_provider_modscope
@Yadnyawalkya please update